contracts/strategies/allocation/donation-voting-open-offchain-calc/README.md

```diff
@@ -14,8 +14,10 @@ At time of writing (June 2023), most prominent QF formulas are too computational
 ### Recipient logic
 In this strategy, recipients must submit an application and be approved by pool admins. 
 - **Recipient information**
-    - In this strategy, recipients must provide a recipientID and a payout address at which they can receive funds. 
-        - If the Allo registry setting is required, then the Allo identityID is used as the recipientID
+    - prospective recipients must provide an address where they can receive funds
+    - prospective recipients must provide a `recipientId` to be the unique identifier for their application
```

Comment:
should we be forcing the `address` type for this value?

Reply:
Yes, good call out. Updated